jaus_ros_bridge/src/JausDataManager.cpp

```diff
@@ -152,36 +124,20 @@ void JausDataManager::ProcessReceivedData(char* buffer)
     ResetAll();
   }
   else if (strcmp(buffer, DISCONNECT_COMMAND) == 0) {
-    jaus_ros_bridge::ActivateManualControl msg;
-    msg.activate_manual_control = false;
-    _publisher_ActivateManualControl.publish(msg);
     ROS_INFO(DISCONNECT_COMMAND);
   }
   else if (strcmp(buffer, ACTIVATE_MAUNAL_CONTROL) == 0) {
-    jaus_ros_bridge::ActivateManualControl msg;
-    msg.activate_manual_control = true;
-    _publisher_ActivateManualControl.publish(msg);
-
-    if(!_activateManualControlEnabled&&_deactivateFromOCU){
+      _missionCommands.StopMission();
       ROS_INFO(ACTIVATE_MAUNAL_CONTROL);
-      _deactivateFromOCU = false;
-    }
-    
-    _activateManualControlEnabled = true;
-    //_needTimerUpdate = true;
+      _finControl.PublishFinAngles(true);
+      _thrusterControl.PublishRPM(true);
```

Comment:
@patapufate we made a mistake here. Dropping manual control timeouts in favor of the mux makes sense so long as the bridge just bridges. This is holding state. If we the connection with the OCU is lost, the bridge will keep publishing the last message and the mux won't ever switch back.